applications/StructuralMechanicsApplication/tests/shell_test/Shell_T3_Thick_linear_static_test_materials.json

```diff
@@ -0,0 +1,19 @@
+{
+    "properties": [{
+            "model_part_name": "Parts_surface",
+            "properties_id": 1,
+            "Material": {
+                    "name": "Material",
+                    "constitutive_law": {
+                            "name": "KratosMultiphysics.StructuralMechanicsApplication.LinearElasticPlaneStress2DLaw"
+                    },
```

Comment:
Avoid the kratosmultiphysics.  Here

Reply:
Removing it didn't work!
Neither from the variable names nor from the CL law!
Do you have an idea why? Also other files use `KratosMultiphysics. ...`
I can fix it in a separate PR